lib/node_modules/@stdlib/repl/lib/output_stream.js

```diff
@@ -285,8 +285,8 @@ setNonEnumerableReadOnly( OutputStream.prototype, '_pagerHeight', function pager
 * @returns {boolean} boolean indicating whether content is "scrollable"
 */
 setNonEnumerableReadOnly( OutputStream.prototype, '_isScrollable', function isScrollable( N ) {
-	var h = this._pagerHeight();
-	return ( h > 0 && N > h );
+	var vh = this._repl.viewportHeight();
+	return ( vh >= MIN_VIEWPORT_HEIGHT && N > vh - 2 ); // 2 rows reserved for the input and the next prompt
```

Comment:
@Snehil-Shah Why only `N > vh-2`? Don't we need to avoid for the pager UI, as well (i.e., the footer)? If so, then `this._pagerHeight()` would seem correct.

Reply:
This is the constraint for when there is a need for scrolling. There is no pager UI when we don't need scrolling, if the output can fit the entire screen (excluding the prompts) there should be no need of paging or accounting for the pager UI

Reply:
Right. Okay. Makes sense.